readme.md

```diff
@@ -103,17 +103,17 @@ For details on OS support see the following resources:
-|---------------------------------------------------------------------------------------------------------------|---------------------     |-------------------------|--------------------------------------------------------------------------------------------------------------------------|
-| [Identity registry (CRUD)](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry) | :heavy_check_mark:        | HTTP | Use your backend app to perform CRUD operation for individual device or in bulk.                                         |
-| [Cloud-to-device messaging](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-c2d)     | :heavy_check_mark:        | AMQP | Use your backend app to send cloud-to-device messages in AMQP and AMQP-WS, and set up cloud-to-device message receivers. |
-| [Direct Methods operations](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods)   | :heavy_check_mark:        | HTTP | Use your backend app to invoke direct method on device.                                                                  |
-| [Device Twins operations](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-device-twins)       | :heavy_check_mark:        | HTTP | Use your backend app to perform twin operations.                        |
-| [Query](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-query-language)                       | :heavy_check_mark:        | HTTP | Use your backend app to perform query for information.                                                                   |
-| [Jobs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs)                                  | :heavy_check_mark:        | HTTP | Use your backend app to perform job operation.                                                                           |
-| [File Upload](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-file-upload)                    | :heavy_check_mark:        | AMQP | Set up your backend app to receive file upload notifications. 
-| [Digital Twin Client](https://docs.microsoft.com/en-us/azure/iot-pnp/overview-iot-plug-and-play)              | :heavy_check_mark:        | HTTP | Set up your backend app to perform operations on plug and play devices.                                                      |
-| [IoT Hub Statistics](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-metrics)                          | :heavy_check_mark:        | HTTP | Get IoT Hub identity registry statistics; such as total device count for device statistics, and connected device count for service statistics. |
+| Features                                                                                                      | Support                  | Transport protocol used underneath | Client to use | Description                                                                                                              |
```

Comment:
Github shows it very differently in the PR, makes it hard to review. But I really appreciate you adding these changes! Super helpful for customers.

Reply:
Yes and I don't know how to make that better.

Reply:
You can see the rich diff in github which makes it much better.